doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -407,6 +407,8 @@ Cellular samples (renamed from nRF9160 samples)
 
     * An overlay for using DTLS Connection Identifier.
       This significantly reduces the DTLS handshake overhead when doing the LwM2M Update operation.
+    * Support for Hosting MCUmgr client for external MCU. New overlay for enable feature and Device
+      tree overlays for UART1, UART2 and MCUBoot recovery mode.
```

Comment:
```suggestion
    * Support for hosting MCUmgr client in an external MCU.
      Including a new overlay file for enabling this and devicetree overlay files for UART1, UART2, and MCUBboot recovery mode.
```

Reply:
We need to remove that UART1. It is not valid